schemas/core/components/configurator.json

```diff
@@ -89,13 +90,47 @@
     "customerSelection": {
       "type": "object",
       "patternProperties": {
-        "^[a-z0-9\\-]+$": {
-          "type": "array",
-          "items": {
-            "type": "string"
-          }
+        "^.*$": {
```

Comment:
This will match also special characters. Best to stick to only lowercase characters and numbers

Reply:
Sticking to lowercase will not allow using camel case notation which we are currently using for the property names. If this should be further limited from what json allows then we can include lowercase and uppercase but I'm not sure if that is necessary.

Reply:
True i didn't thinking about lowercase. Still we don't really hope to receive non-alphabetical characters, so it should be forbidden

Reply:
Change to allow only alphanumerical including capital cases